src/vm/memory/relocatable.zig

```diff
@@ -255,6 +262,18 @@ pub const MaybeRelocatable = union(enum) {
     relocatable: Relocatable,
     felt: Felt252,
 
+    pub fn getFelt(self: *const Self) ?Felt252 {
+        switch (self.*) {
+            .felt => |felt| return felt,
+            else => return null,
+        }
+    }
+    pub fn getRelocatable(self: *const Self) ?Relocatable {
```

Comment:
We already have a `tryIntoRelocatable` method here (not fully satisfied by the naming, should be renamed `toRelocatable` to fulfill Ziglang naming convention as long as we must use a `try` in front of it). But apart from the fact that we return a `null` here in the case where the type is not satisfied, which is not necessarily a good thing, it is better to return an error in my opinion. It seems to me that this function duplicates the other previously created, right?